dev-proxy/ProxyHost.cs

```diff
@@ -346,9 +346,65 @@ public RootCommand GetRootCommand(ILogger logger)
         var outdatedShortOption = new Option<bool>("--short", "Return version only");
         outdatedCommand.AddOption(outdatedShortOption);
         outdatedCommand.SetHandler(async versionOnly => await OutdatedCommandHandler.CheckVersionAsync(versionOnly, logger), outdatedShortOption);
-
         command.Add(outdatedCommand);
 
+        var jwtCommand = new Command("jwt", "Manage JSON Web Tokens ");
+        var jwtCreateCommand = new Command("create", "Create a new JWT token");
+        var jwtSchemeOption = new Option<string>("--scheme", "The scheme name to use for the token.");
+        jwtSchemeOption.SetDefaultValue("Bearer");
+        jwtCreateCommand.AddOption(jwtSchemeOption);
+        var jwtNameOption = new Option<string>("--name", "The name of the user to create the token for.");
+        jwtNameOption.AddAlias("-n");
+        jwtNameOption.SetDefaultValue("Dev Proxy");
+        jwtCreateCommand.AddOption(jwtNameOption);
+        var jwtAudienceOption = new Option<IEnumerable<string>>("--audience", "The audiences to create the token for.")
+        {
+            AllowMultipleArgumentsPerToken = true
+        };
+        jwtAudienceOption.AddAlias("-a");
+        jwtAudienceOption.SetDefaultValue("https://myserver.com");
+        jwtCreateCommand.AddOption(jwtAudienceOption);
+        var jwtIssuerOption = new Option<string>("--issuer", "The issuer of the token.");
+        jwtIssuerOption.AddAlias("-i");
+        jwtIssuerOption.SetDefaultValue("dev-proxy");
+        jwtCreateCommand.AddOption(jwtIssuerOption);
+        var jwtRolesOption = new Option<IEnumerable<string>>("--roles", "A role claim to add to the token. Specify once for each scope. Use a comma to separate scopes.")
+        {
+            AllowMultipleArgumentsPerToken = true
+        };
+        jwtRolesOption.AddAlias("-r");
+        jwtCreateCommand.AddOption(jwtRolesOption);
+        var jwtScopesOption = new Option<IEnumerable<string>>("--scopes", "A scope claim to add to the token. Specify once for each scope. Use a comma to separate roles.")
+        {
+            AllowMultipleArgumentsPerToken = true
+        };
+        jwtScopesOption.AddAlias("-s");
+        jwtCreateCommand.AddOption(jwtScopesOption);
+        var jwtClaimsOption = new Option<IEnumerable<string>>("--claims", "Claims to add to the token. Specify once for each claim in the format \"name=value\". Use a comma to separate claims.")
+        {
+            AllowMultipleArgumentsPerToken = true
```

Comment:
Despite the setting, I'm getting an error specifying multiple claims:

<img width="1058" alt="image" src="https://github.com/user-attachments/assets/43e801df-0f76-499d-aae9-b333aff3f403">

Seems like the shorthand `-c` collides with the global option for a config file. Let's remove it here.

<img width="1062" alt="image" src="https://github.com/user-attachments/assets/88d634da-e49c-4547-8b8f-367b0fa8ee22">